packages/smooth_app/lib/pages/scan/search_page.dart

```diff
@@ -174,9 +174,11 @@ class _SearchFieldState extends State<SearchField> {
   }
 
   void _handleTextChange() {
-    setState(() {
-      _isEmpty = _textController.text.isEmpty;
-    });
+    if (_textController.text.isEmpty || _textController.text.length == 1) {
```

Comment:
took me a few moments to realize why you did this, can you add a comment

Reply:
Added a comment